webclient/pas/views.py

```diff
@@ -857,6 +857,30 @@ def get_queryset(self):
         )
         return self.check_filter_permission(qs).order_by("id")
     
+    def check_filter_permission(self, qs):  
```

Comment:
@jnihnat Nemůže nějak ta změna ovlivnit i jiné využití funkce na: https://github.com/ARUP-CAS/aiscr-webamcr/blob/b991dc83bc54653673ee697caf10aa19d47b83b6/webclient/pas/views.py#L681 ?

Možná je to ok, ale jen se raději ujišťuji, že jak jste to dělal v rychlosti, tak jste s tím počítal...

Reply:
Vysvětleno s @jhavrlant - už jsem to pochopil, vše je ok, beru zpět.